<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-DPRKKQA" failures="0" tests="2" name="apiNotRun.XmlPayload" time="20.731" errors="2" timestamp="2024-01-11T08:14:41 IST" skipped="0">
  <testcase classname="apiNotRun.XmlPayload" name="putRequest" time="19.582">
    <error message="Failed to parse the XML document" type="io.restassured.path.xml.exception.XmlPathException">
      <![CDATA[io.restassured.path.xml.exception.XmlPathException: Failed to parse the XML document
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1039)
at io.restassured.path.xml.XmlPath.getFromPath(XmlPath.java:423)
at io.restassured.path.xml.XmlPath.get(XmlPath.java:317)
at io.restassured.path.xml.XmlPath.getString(XmlPath.java:571)
at apiNotRun.XmlPayload.putRequest(XmlPayload.java:83)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1251)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:637)
at groovy.xml.XmlSlurper.parse(XmlSlurper.java:199)
at groovy.xml.XmlSlurper.parse(XmlSlurper.java:252)
at groovy.xml.XmlSlurper.parseText(XmlSlurper.java:282)
at io.restassured.path.xml.XmlPath$1.method(XmlPath.java:840)
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1032)
... 30 more
]]>
    </error>
  </testcase> <!-- putRequest -->
  <testcase classname="apiNotRun.XmlPayload" name="postRequest" time="1.149">
    <error message="Failed to parse the XML document" type="io.restassured.path.xml.exception.XmlPathException">
      <![CDATA[io.restassured.path.xml.exception.XmlPathException: Failed to parse the XML document
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1039)
at io.restassured.path.xml.XmlPath.getFromPath(XmlPath.java:423)
at io.restassured.path.xml.XmlPath.get(XmlPath.java:317)
at io.restassured.path.xml.XmlPath.getString(XmlPath.java:571)
at apiNotRun.XmlPayload.postRequest(XmlPayload.java:54)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1251)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:637)
at groovy.xml.XmlSlurper.parse(XmlSlurper.java:199)
at groovy.xml.XmlSlurper.parse(XmlSlurper.java:252)
at groovy.xml.XmlSlurper.parseText(XmlSlurper.java:282)
at io.restassured.path.xml.XmlPath$1.method(XmlPath.java:840)
at io.restassured.path.xml.XmlPath$LazyXmlParser.invoke(XmlPath.java:1032)
... 30 more
]]>
    </error>
  </testcase> <!-- postRequest -->
</testsuite> <!-- apiNotRun.XmlPayload -->
